### ISSUE TITLE
Node v18 

### ISSUE THREAD

**Issue by victors1681** [ayqy/string-to-file-stream#8]
Hey @ayqy  I was using this package in node v16.x and now upgrading my app I'm getting the following error: 

<img width="589" alt="Screenshot 2023-09-10 at 6 20 30 PM" src="https://github.com/ayqy/string-to-file-stream/assets/11906852/cd80db0c-f100-45ea-9d15-2d2419d7b02f">

Seems `closed` in Readable is a real property 

<img width="541" alt="Screenshot 2023-09-10 at 6 19 57 PM" src="https://github.com/ayqy/string-to-file-stream/assets/11906852/50f2e545-f258-452a-9c34-e81f629648b1">

When I comment the initialization of the `closed` property works as expected

<img width="684" alt="Screenshot 2023-09-10 at 6 23 42 PM" src="https://github.com/ayqy/string-to-file-stream/assets/11906852/91a376d2-33aa-4ce9-9214-03b5d5aae07f">

<img width="417" alt="Screenshot 2023-09-10 at 6 24 55 PM" src="https://github.com/ayqy/string-to-file-stream/assets/11906852/d4fcd0ed-848e-4b43-958e-73aa0f22a352">

thank you for this package

**Comment by ayqy** [comment]
Maybe a breaking change if someone read `closed` property on Node.js v18- like this
```
const string2fileStream = require('string-to-file-stream');
const assert = require('assert');

const input = 'Oh, my great data!';
const s = string2fileStream(input);
s.on('data', (chunk) => {
  assert.equal(chunk.toString(), input);
  setTimeout(() => {
    // got undefined after closed property was commented on local Node.js v14.17.4
    console.log(s.closed);
  }, 3000);
});
```

**Comment by ayqy** [comment]
Merged and release 2.0.0.

**Comment by victors1681** [comment]
nice! thank you @ayqy!
